pkg/util/mount/mount.go

```diff
@@ -49,6 +52,66 @@ type IMount interface {
 	GetInstanceID() (string, error)
 	MakeFile(pathname string) error
 	MakeDir(pathname string) error
+	PathExists(path string) (bool, error)
+	GetDeviceStats(path string) (*DeviceStats, error)
+}
+
+type DeviceStats struct {
+	Block bool
+
+	AvailableBytes  int64
+	TotalBytes      int64
+	UsedBytes       int64
+	AvailableInodes int64
+	TotalInodes     int64
+	UsedInodes      int64
+}
+
+func (m *Mount) PathExists(volumePath string) (bool, error) {
```

Comment:
:nit: i think this func exists in k8s.io/util/

Reply:
It has an additional check for a "corrupted mount". I'm not sure that this is required and this may introduce a potential false-positive.

https://github.com/kubernetes/utils/blob/d1ab8797c55812f4fefe2c7b00a0d04a4740a93c/mount/mount_helper_common.go#L99-L100